src/components/protocol_handler/src/protocol_payload.cc

```diff
@@ -93,6 +93,7 @@ std::ostream& operator<<(std::ostream& os, const ProtocolPayloadV2& payload) {
             << ", data (bytes): " << payload.data.size() << ")";
 }
 
+// cppcheck-suppress unusedFunction //Used in unit tests
 size_t ProtocolPayloadV2SizeBits() {
```

Comment:
Could this be wrapped in `#ifdef BUILD_TESTS`?

Reply:
Done in 9f1a645791e598169044d384c2a131addb74c075